gensim/models/wrappers/dtmmodel.py

```diff
@@ -60,32 +63,40 @@ def __init__(self, dtm_path, corpus=None, time_slices=None, mode='fit', model='d
                  id2word=None, prefix=None, lda_sequence_min_iter=6, lda_sequence_max_iter=20, lda_max_em_iter=10,
                  alpha=0.01, top_chain_var=0.005, rng_seed=0, initialize_lda=True):
         """
+
         Parameters
         ----------
         dtm_path : str
-            Path to the dtm executable,  e.g. `C:/dtm/dtm-win64.exe`.
+            Path to the dtm binary, e.g. `/home/username/dtm/dtm/main`.
```

Comment:
-1: the Windows example was better. Windows users are generally more clueless; Linux ppl can easily do the mental bridge between Windows `.exe` => Linux binary, but Windows users can't.

Or just simply use both examples.